packages/client/src/comp/Address/Fields.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import Input from 'muicss/lib/react/input'
+import styled from 'styled-components'
+import { BaseInput } from '../util/Input'
+import { Select, Option } from 'muicss/react'
+import { useControlRef } from '../util/ControlRef'
+import { RoundedButton } from '../util/Button'
+import { allStates } from '../../common'
+import { useAppHistory } from '../../lib/path'
+import { AddressInputPartContainer } from './Container'
+import { FetchingDataContainer } from '../../lib/unstated'
+
+interface Props {
+  type: 'initialAddress' | 'separateMailing'
+}
+
+const FlexBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-flow: row wrap;
+  align-items: center;
+  margin: 1em 0;
+`
+
+const Flex = styled.div<{ basis?: string }>`
+  flex-basis: ${p => p.basis ?? '100%'};
+  /* Fixes needless scrollbars showing on Linux/Windows */
+  .mui-select label { overflow: hidden; }
+`
+
+// Since react-hooks/exhaustive-deps will trigger re-renders due to its
+// depedencies, we declare this function outside of our component.
+const useDidMount = (fun: () => void) => React.useEffect(fun, [])
+
+export const AddressFields: React.FC<Props> = ({ type }) => {
```

Comment:
Let's rename this `Address/Input.tsx` and `AddressFields` to `AddressInput`-- since it's actually a replacement input (like a date chooser)